pacts/cli-lobby.json

```diff
@@ -0,0 +1,48 @@
+{
+  "consumer": {
+    "name": "CLI"
+  },
+  "provider": {
+    "name": "Lobby"
+  },
+  "interactions": [
+    {
+      "description": "A request to create a lobby",
+      "providerState": "The server is online",
+      "request": {
+        "method": "POST",
+        "path": "/lobby",
+        "headers": {
+          "Content-Type": "application/json"
+        }
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": {
+          "game-id": 1,
+          "lobby-status": "\"string\"",
+          "player-id": "\"string\""
+        },
+        "matchingRules": {
```

Comment:
Here to say you only have to match the type of variable?

Reply:
Yes, otherwise it would fail if the game-id weren't exactly the number 1 every time... which is definitely **not** what we want.